apps/vr-tests-react-components/package.json

```diff
@@ -49,6 +49,7 @@
     "@fluentui/react-progress": "*",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30414&commitId=41f9023202627dacd80f4405fb9f97d3176c3d78&env=prod&clientType=FLUENTUI)


  <details><summary><b>Rating Converged</b> 405 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Rating Converged.Brand Rating size extra large filled - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large filled - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large filled - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large filled - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large filled - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large filled - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large filled.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large filled.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large filled.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large no value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large no value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large no value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large no value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large no value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large no value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large no value.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large no value.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large no value.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large with value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large with value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large with value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large with value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large with value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large with value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large with value.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large with value.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size extra large with value.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size large filled - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size large filled - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size large filled - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size large filled - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size large filled - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size large filled - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size large filled.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size large filled.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size large filled.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size large no value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size large no value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size large no value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size large no value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size large no value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size large no value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size large no value.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size large no value.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size large no value.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size large with value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size large with value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size large with value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size large with value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size large with value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size large with value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size large with value.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size large with value.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size large with value.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium filled - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium filled - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium filled - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium filled - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium filled - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium filled - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium filled.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium filled.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium filled.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium no value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium no value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium no value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium no value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium no value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium no value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium no value.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium no value.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium no value.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium with value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium with value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium with value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium with value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium with value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium with value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium with value.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium with value.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size medium with value.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size small filled - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size small filled - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size small filled - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size small filled - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size small filled - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size small filled - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size small filled.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size small filled.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size small filled.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size small no value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size small no value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size small no value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size small no value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size small no value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size small no value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size small no value.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size small no value.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size small no value.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size small with value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size small with value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size small with value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size small with value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size small with value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size small with value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating size small with value.active.chromium.png|0| Added|
|Rating Converged.Brand Rating size small with value.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating size small with value.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating with custom circle icons - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Brand Rating with custom circle icons - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating with custom circle icons - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating with custom circle icons - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Brand Rating with custom circle icons - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating with custom circle icons - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating with custom circle icons.active.chromium.png|0| Added|
|Rating Converged.Brand Rating with custom circle icons.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating with custom circle icons.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating with custom square icons - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Brand Rating with custom square icons - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating with custom square icons - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating with custom square icons - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Brand Rating with custom square icons - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating with custom square icons - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating with custom square icons.active.chromium.png|0| Added|
|Rating Converged.Brand Rating with custom square icons.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating with custom square icons.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating with half value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Brand Rating with half value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating with half value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating with half value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Brand Rating with half value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating with half value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Brand Rating with half value.active.chromium.png|0| Added|
|Rating Converged.Brand Rating with half value.hover.chromium.png|0| Added|
|Rating Converged.Brand Rating with half value.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large filled - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large filled - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large filled - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large filled - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large filled - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large filled - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large filled.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large filled.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large filled.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large no value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large no value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large no value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large no value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large no value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large no value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large no value.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large no value.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large no value.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large with value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large with value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large with value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large with value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large with value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large with value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large with value.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large with value.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size extra large with value.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large filled - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large filled - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large filled - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large filled - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large filled - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large filled - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large filled.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large filled.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large filled.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large no value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large no value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large no value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large no value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large no value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large no value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large no value.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large no value.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large no value.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large with value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large with value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large with value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large with value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large with value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large with value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large with value.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large with value.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size large with value.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium filled - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium filled - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium filled - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium filled - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium filled - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium filled - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium filled.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium filled.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium filled.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium no value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium no value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium no value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium no value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium no value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium no value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium no value.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium no value.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium no value.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium with value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium with value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium with value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium with value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium with value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium with value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium with value.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium with value.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size medium with value.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small filled - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small filled - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small filled - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small filled - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small filled - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small filled - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small filled.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small filled.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small filled.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small no value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small no value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small no value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small no value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small no value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small no value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small no value.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small no value.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small no value.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small with value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small with value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small with value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small with value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small with value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small with value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small with value.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small with value.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating size small with value.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating with custom circle icons - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating with custom circle icons - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating with custom circle icons - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating with custom circle icons - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating with custom circle icons - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating with custom circle icons - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating with custom circle icons.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating with custom circle icons.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating with custom circle icons.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating with custom square icons - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating with custom square icons - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating with custom square icons - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating with custom square icons - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating with custom square icons - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating with custom square icons - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating with custom square icons.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating with custom square icons.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating with custom square icons.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating with half value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating with half value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating with half value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating with half value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating with half value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating with half value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Marigold Rating with half value.active.chromium.png|0| Added|
|Rating Converged.Marigold Rating with half value.hover.chromium.png|0| Added|
|Rating Converged.Marigold Rating with half value.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large filled - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large filled - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large filled - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large filled - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large filled - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large filled - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large filled.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large filled.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large filled.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large no value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large no value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large no value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large no value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large no value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large no value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large no value.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large no value.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large no value.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large with value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large with value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large with value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large with value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large with value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large with value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large with value.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large with value.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size extra large with value.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large filled - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large filled - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large filled - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large filled - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large filled - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large filled - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large filled.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large filled.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large filled.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large no value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large no value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large no value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large no value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large no value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large no value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large no value.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large no value.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large no value.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large with value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large with value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large with value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large with value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large with value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large with value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large with value.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large with value.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size large with value.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium filled - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium filled - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium filled - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium filled - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium filled - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium filled - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium filled.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium filled.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium filled.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium no value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium no value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium no value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium no value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium no value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium no value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium no value.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium no value.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium no value.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium with value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium with value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium with value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium with value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium with value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium with value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium with value.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium with value.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size medium with value.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small filled - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small filled - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small filled - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small filled - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small filled - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small filled - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small filled.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small filled.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small filled.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small no value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small no value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small no value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small no value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small no value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small no value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small no value.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small no value.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small no value.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small with value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small with value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small with value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small with value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small with value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small with value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small with value.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small with value.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating size small with value.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating with custom circle icons - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating with custom circle icons - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating with custom circle icons - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating with custom circle icons - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating with custom circle icons - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating with custom circle icons - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating with custom circle icons.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating with custom circle icons.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating with custom circle icons.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating with custom square icons - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating with custom square icons - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating with custom square icons - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating with custom square icons - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating with custom square icons - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating with custom square icons - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating with custom square icons.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating with custom square icons.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating with custom square icons.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating with half value - Dark Mode.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating with half value - Dark Mode.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating with half value - Dark Mode.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating with half value - High Contrast.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating with half value - High Contrast.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating with half value - High Contrast.rest.chromium.png|0| Added|
|Rating Converged.Neutral Rating with half value.active.chromium.png|0| Added|
|Rating Converged.Neutral Rating with half value.hover.chromium.png|0| Added|
|Rating Converged.Neutral Rating with half value.rest.chromium.png|0| Added|

</details>


  <details><summary><b>RatingDisplay Converged</b> 108 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|RatingDisplay Converged.Brand RatingDisplay size extra large with value - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size extra large with value - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size extra large with value and count - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size extra large with value and count - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size extra large with value and count.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size extra large with value.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size large with value - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size large with value - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size large with value and count - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size large with value and count - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size large with value and count.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size large with value.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size medium with value - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size medium with value - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size medium with value and count - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size medium with value and count - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size medium with value and count.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size medium with value.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size small with value - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size small with value - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size small with value and count - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size small with value and count - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size small with value and count.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay size small with value.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay with custom circle icons - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay with custom circle icons - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay with custom circle icons.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay with custom square icons - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay with custom square icons - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay with custom square icons.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay with half value - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay with half value - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Brand RatingDisplay with half value.chromium.png|0| Added|
|RatingDisplay Converged.Compact Brand RatingDisplay - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Compact Brand RatingDisplay - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Compact Brand RatingDisplay.chromium.png|0| Added|
|RatingDisplay Converged.Compact Marigold RatingDisplay - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Compact Marigold RatingDisplay - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Compact Marigold RatingDisplay.chromium.png|0| Added|
|RatingDisplay Converged.Compact Neutral RatingDisplay - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Compact Neutral RatingDisplay - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Compact Neutral RatingDisplay.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size extra large with value - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size extra large with value - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size extra large with value and count - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size extra large with value and count - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size extra large with value and count.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size extra large with value.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size large with value - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size large with value - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size large with value and count - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size large with value and count - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size large with value and count.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size large with value.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size medium with value - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size medium with value - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size medium with value and count - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size medium with value and count - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size medium with value and count.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size medium with value.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size small with value - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size small with value - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size small with value and count - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size small with value and count - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size small with value and count.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay size small with value.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay with custom circle icons - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay with custom circle icons - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay with custom circle icons.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay with custom square icons - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay with custom square icons - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay with custom square icons.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay with half value - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay with half value - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Marigold RatingDisplay with half value.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size extra large with value - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size extra large with value - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size extra large with value and count - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size extra large with value and count - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size extra large with value and count.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size extra large with value.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size large with value - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size large with value - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size large with value and count - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size large with value and count - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size large with value and count.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size large with value.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size medium with value - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size medium with value - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size medium with value and count - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size medium with value and count - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size medium with value and count.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size medium with value.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size small with value - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size small with value - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size small with value and count - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size small with value and count - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size small with value and count.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay size small with value.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay with custom circle icons - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay with custom circle icons - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay with custom circle icons.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay with custom square icons - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay with custom square icons - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay with custom square icons.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay with half value - Dark Mode.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay with half value - High Contrast.chromium.png|0| Added|
|RatingDisplay Converged.Neutral RatingDisplay with half value.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
Looks like the tests are cropping correctly now 🥳

You might want to look into the vertical alignment of the text of RatingDisplay, especially for size small. But it doesn't seem to be a bug in the test, so it is fine for this PR.

One random change in Avatar from failing to load an image is unrelated to these changes and can be ignored.